### PR TITLE
Sentry setup

### PR DESCRIPTION
**Pull request by jasonappah**
## Overview

This PR integrates [Sentry](https://sentry.io), which will allow us to keep tabs on any issues that pop up in production and get detailed logs for debugging issues.

List the GitHub issues containing the issues relevant to this pull request.

[For example, "Resolves #1, closes #2, fixes #3", etc.]

Describe your changes here at a high level, describing how this PR fits into the
rest of the project.

## What Changed

Go more into detail about key files that were modified and why they were
updated. Do not list every file that was modified; only note the ones most
relevant to this feature or bug fix.

## Other Notes

If were roadblocks encountered during development that remain unresolved or any
future additions or changes to make, note them here. Otherwise. Feel free to
delete this section if it isn't needed.

### CODE REVIEW

File: sentry.server.config.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from '@sentry/nextjs';
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
+
+Sentry.init({
+  dsn:
+    SENTRY_DSN ||
+    'https://edb9e327f9024e7599eae859bad9be47@o4504918397353984.ingest.sentry.io/4504924302409728',
```
Comment (ZzRanger) [inline sentry.server.config.js:12]: I didn't read docs but should this be env file or is this fine?

File: sentry.client.config.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+// This file configures the initialization of Sentry on the browser.
+// The config you add here will be used whenever a page is visited.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from '@sentry/nextjs';
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
```
Comment (ZzRanger) [inline sentry.client.config.js:7]: Should probably update .env.example too

**Review by ZzRanger — CHANGES_REQUESTED** [review]
Imma be real idt I'll be able to properly review this until at least Wed-Thurs so if someone else minds looking at sentry before merge that'd be great. PR seems fine to me aside from updating .env.example

File: sentry.server.config.js (hunk 1 above)
Comment (jasonappah) [reply sentry.server.config.js:12]: This should be fine since this ends up being used in the browser anyway

File: sentry.client.config.js (hunk 2 above)
Comment (jasonappah) [reply sentry.client.config.js:7]: will do

**Review by ZzRanger — DISMISSED** [review]
LGTM!

File: sentry.client.config.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Sentry.init({
```
Comment (ZzRanger) [inline sentry.client.config.js:9]: Small thing but should we specify the [environment](https://docs.sentry.io/platforms/javascript/guides/nextjs/configuration/environments/) for Sentry?

Comment (jasonappah) [reply sentry.client.config.js:9]: Can't find the docs page that said this but the environment is automatically set from NODE_ENV

Comment (jasonappah) [reply sentry.client.config.js:9]: Actually i read that from a legacy docs page - lemme fix

Comment (jasonappah) [reply sentry.client.config.js:9]: should be gtg

**Comment by jasonappah** [comment]
Superseded by #578

**Pull request closed without merging**